docs/blog/_posts/2019-03-04-13th-dotty-milestone-release.md

```diff
@@ -0,0 +1,439 @@
+
+- become more opinionated by promoting programming idioms we found to work well,
+- simplify where possible,
+- eliminate inconsistencies and surprising behaviours,
+- build on strong foundations to ensure the design hangs together well,
+- consolidate language constructs to improve the language’s consistency, safety, ergonomics, and
+  performance.
+
+You can learn more about Dotty on our [website](https://dotty.epfl.ch).
+
+<!--more-->
+
+This is our 13th scheduled release according to our
+[6-week release schedule](https://dotty.epfl.ch/docs/usage/version-numbers.html).
+
+# What’s new in the 0.13.0-RC1 technology preview?
+
+## Experimental support for Spark
+
+Dotty projects have always been able to [depend on Scala 2
+libraries](https://github.com/lampepfl/dotty-example-project#getting-your-project-to-compile-with-dotty),
+and this usually works fine as long as the Dotty code does not call a Scala 2
+macro directly. However, [Spark](http://spark.apache.org/) was known to not work
+correctly as it heavily relies on Java serialization which we were not fully
+supporting.
+
+Meanwhile, at EPFL, we've started updating our Scala courses to use Dotty
+instead of Scala 2, the *Functional Programming* course given last semester went
+smoothly, but the *Parallelism and Concurrency* course given in the
+Spring semester teaches Spark, which means we needed to support it in Dotty!
+
+Luckily, this turned out to be mostly straightforward: we adopted the [object
+serialization scheme](https://github.com/lampepfl/dotty/pull/5775) and [lambda
+serialization scheme](https://github.com/lampepfl/dotty/pull/5837) pioneered by
+Scala 2, and that was enough to make our Spark assignments run correctly! This
+doesn't mean that our support is perfect however, so don't hesitate to [open an
+issue](http://github.com/lampepfl/dotty/issues) if something is amiss.
+
+## Introducing top level definitions
+
+_Top level_ definitions are now supported. This means that package objects are
+now redundant, and will be phased out. This means that all kinds of definitions
+can be written at the top level.
+
+```scala
+package p
+
+type Labelled[T] = (String, T)
+
+val a: Labelled[Int] = ("count", 1)
+def b = a._2
+```
+
+You can read about [dropping package
+objects](https://dotty.epfl.ch/docs/reference/dropped-features/package-objects.html)
+at the documentation linked or at the relevant PR
+[#5754](https://github.com/lampepfl/dotty/pull/5754).
+
+## All things impl... implied
+
+Scala's implicits are its most distinguished feature. They are _the_ fundamental
+way to abstract over context. They represent a single concept with an extremely
+varied number of use cases, among them: implementing type classes, establishing
+context, dependency injection, expressing capabilities, computing new types and
+proving relationships between them.
+
+However, we identify a few consequences that implicits gave rise to, as a

```

Comment:
This use of the present tense ("we identify") is rather weird, especially in conjunction with the past tense for what it is that's identified ("gave rise to"). Why not say everything in the past tense? "_We identified ...._"